packages/ui/src/components/AccountDisplay.tsx

```diff
@@ -1,29 +1,43 @@
-import { Box } from "@mui/material";
-import { useEffect, useMemo, useState } from "react";
-import { styled } from "@mui/material/styles";
-import { useAccountNames } from "../contexts/AccountNamesContext";
-import { AccountBadge } from "../types";
-import { getDisplayAddress } from "../utils";
-import IdenticonBadge from "./IdenticonBadge";
-import { useApi } from "../contexts/ApiContext";
-import { DeriveAccountInfo, DeriveAccountRegistration } from '@polkadot/api-derive/types';
-import IdentityIcon from "./IdentityIcon";
-import { useGetBalance } from "../hooks/useGetBalance";
+import { Box } from '@mui/material';
+import { useEffect, useMemo, useState } from 'react';
+import { styled } from '@mui/material/styles';
+import { useAccountNames } from '../contexts/AccountNamesContext';
+import { AccountBadge } from '../types';
+import { getDisplayAddress } from '../utils';
+import IdenticonBadge from './IdenticonBadge';
+import { useApi } from '../contexts/ApiContext';
+import {
```

Comment:
This is the case I mentioned, where I would love to have an exception for import. Making those import on several line doesn't add any value in IMHO